### PR TITLE
Persons modal style updates

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

Screen Shot 2021-07-08 at 9 53 34 AM

Screen Shot 2021-07-08 at 9 54 38 AM

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

**Comment by liyiy** [comment]
@clarkus I kind of just snuck the property details expansion back in there but I don't have the best eye for design, let me know if I can clean up anything around it 😄

**Comment by paolodamico** [comment]
Wow this is looking amazing so far, great stuff @liyiy !

**Review by clarkus — CHANGES_REQUESTED** [review]
This is looking great. I can see a few places where things could be aligned with other features. I know you're still working on this, but are you looking for feedback on the other search and filtering options? Are we still allowing for lists to be saved as new cohorts?

**Let's reuse the expand / collapse component from sessions here.** What you have works, but if we can reuse the same components from other areas that would help users build familiarity:
Screen Shot 2021-07-08 at 11 48 13 AM

**Can we make the property list full width?** The column of names is going to contain the greatest amount of variability, so we could just increase the width of that column to fill the available space. What do you think?

**Comment by liyiy** [comment]
> This is looking great. I can see a few places where things could be aligned with other features. I know you're still working on this, but are you looking for feedback on the other search and filtering options? Are we still allowing for lists to be saved as new cohorts?
> 
> **Let's reuse the expand / collapse component from sessions here.** What you have works, but if we can reuse the same components from other areas that would help users build familiarity:
> Screen Shot 2021-07-08 at 11 48 13 AM
> 
> **Can we make the property list full width?** The column of names is going to contain the greatest amount of variability, so we could just increase the width of that column to fill the available space. What do you think?

Yes (the cohort button was there, just hidden behind a feature flag in my screenshot). I've also updated the expand icon, didn't know we built our own custom one, cool! 

As for the search bar, we're tabling that for now due to some querying complications. For the sessions filter, I haven't looked into that too much yet so that's still a TODO as I'm currently working on the time conversion histogram, perhaps we can come back to that when Alex is back? 

Screen Shot 2021-07-08 at 4 05 25 PM

**Comment by liyiy** [comment]
Will merge unless I'm missing anything important in this iteration :D

File: frontend/src/scenes/trends/PersonModal.tsx [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
-import { PersonsTable } from 'scenes/persons/PersonsTable'
+import { deepLinkToPersonSessions } from 'scenes/persons/PersonsTable'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 import { ViewType } from 'scenes/insights/insightLogic'
 import { ActionFilter, EntityTypes, EventPropertyFilter, FilterType, SessionsPropertyFilter } from '~/types'
 import { ACTION_TYPE, EVENT_TYPE } from 'lib/constants'
 import { personsModalLogic } from './personsModalLogic'
-import { funnelLogic } from 'scenes/funnels/funnelLogic'
-
+import Col from 'antd/es/grid/col'
```
Comment (paolodamico) [inline frontend/src/scenes/trends/PersonModal.tsx:13]: Nit: better to import from `ant` directly

File: frontend/src/scenes/trends/PersonModal.tsx [hunk 2]
```diff
@@ -96,67 +125,45 @@ export function PersonModal({ visible, view, onSaveCohort }: Props): JSX.Element
                                 flexDirection: 'column',
                                 width: '100%',
                                 alignItems: 'flex-start',
+                                padding: '0px 16px',
                             }}
                         >
-                            <span style={{ paddingBottom: 12 }}>
+                            {featureFlags['persons-modal-search'] && (
```
Comment (paolodamico) [inline frontend/src/scenes/trends/PersonModal.tsx:131]: Using hard-coded `persons-modal-search` instead of the declared constant

**Review by paolodamico — APPROVED** [review]
lgtm! minor comments, we can iterate from here too!

**Pull request merged**